docs/markdown/Using Pants/environments.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "Environments: Cross-Platform or Remote Builds"
+slug: "environments"
+hidden: false
+createdAt: "2022-10-03T21:39:51.235Z"
+updatedAt: "2022-10-03T21:39:51.235Z"
+---
+Environments
+============
+
+By default, Pants will execute all sandboxed build work directly on localhost. But defining and using additional "environments" for particular targets allows Pants to transparently execute some or all of your build either:
+1. locally in Docker containers
+2. remotely via [remote execution](doc:remote-execution)
+3. locally, but with a non-default set of environment variables and settings (such as for cross-building)
```

Comment:
Another important use case is simply that macOS and Linux need different config. Environments can still be useful even without the 3 above use cases